aws/elasticache.go

```diff
@@ -70,21 +172,16 @@ func nukeAllElasticacheClusters(session *session.Session, clusterIds []*string)
 		}
 	}
 
-	if len(deletedClusterIds) > 0 {
-		logging.Logger.Infof("Confirming deletion of %d Elasticache clusters in region %s", len(deletedClusterIds), *session.Config.Region)
+	logging.Logger.Infof("[OK] %d Elasticache clusters deleted in %s", len(deletedClusterIds), *session.Config.Region)
+	return nil
+}
 
-		for _, clusterId := range deletedClusterIds {
-			params := elasticache.DescribeCacheClustersInput{
-				CacheClusterId: clusterId,
-			}
+// Custome errors
```

Comment:
```suggestion
// Custom errors
```

Reply:
Ah thanks! I'll open a subsequent PR for the typo.

Reply:
Done here: https://github.com/gruntwork-io/cloud-nuke/pull/336